src/views/docs-view/index.tsx

```diff
@@ -56,14 +58,27 @@ const productsToPrimitives: ProductsToPrimitivesMap = {
 	waypoint: { NestedNode, Placement },
 }
 
-const DocsView = ({ mdxSource, lazy, hideSearch = false }: DocsViewProps) => {
+const layouts = {
+	'docs-root-landing': dynamic(() => import('./components/docs-root-landing')),
+}
+
+const DocsView = ({
+	metadata,
+	mdxSource,
+	lazy,
+	hideSearch = false,
+}: DocsViewProps) => {
 	const currentProduct = useCurrentProduct()
 	const { compiledSource, scope } = mdxSource
 	const additionalComponents = productsToPrimitives[currentProduct.slug] || {}
 	const components = defaultMdxComponents({ additionalComponents })
 	const shouldRenderSearch =
 		!hideSearch && __config.flags.enable_product_docs_search
 
+	const Layout =
+		layouts[metadata?.layout?.name] ??
+		(({ children }) => <div className={s.mdxContent}>{children}</div>)
```

Comment:
No changes here relative to #733, except the use of `s.mdxContent` for spacing. Comment on `docs-view.module.css` has a bit more detail on spacing thoughts.

Reply:
suggestion: let's pull the default layout declaration out of the component body, otherwise re-renders of the view will cause the layout component to re-mount.

```ts
const DefaultLayout = (({ children }) => <div className={s.mdxContent}>{children}</div>
```

Reply:
🙌  ✅ thanks for catching this, updated!